app/src/main/java/de/danoeh/antennapod/ui/screen/home/sections/EpisodesSurpriseSection.java

```diff
@@ -149,6 +153,11 @@ private void loadItems() {
                     this.episodes = episodes;
                     listAdapter.setDummyViews(0);
                     listAdapter.updateData(episodes);
+
+                    boolean isShuffleable = episodes.size() > 1;
```

Comment:
We only load one random episode per subscription. So if there is just one subscription, there is always just one item. This would hide the button while there is actually something to shuffle. I would just show the empty view when there are actually no episodes.